helm_deploy/hmpps-manage-adjudications-api/templates/housekeeping-cronjob.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1beta1
```

Comment:
Taken from the sqs stuff - should work but will require a bit of testing in the dev environment most likely